relayer/src/util/retry.rs

```diff
@@ -2,9 +2,33 @@ use std::time::Duration;
 
 pub use retry::{
     delay::{Fibonacci, Fixed},
-    retry_with_index, OperationResult as RetryResult,
+    retry_with_index, Error as RetryError, OperationResult as RetryResult,
 };
 
+pub trait ToRetryResult<T, E> {
```

Comment:
Yeah, I suppose we can add it back later when we implement the RetryableError logic. Right now since all errors are being retried, this is not really needed.